command_test.go

```diff
@@ -131,6 +131,15 @@ func TestCommand(t *testing.T) {
 
 	})
 
+	t.Run("Run NewSubCommandInheritFlags()", func(t *testing.T) {
+		c.NewSubCommandInheritFlags("name", "description")
+	})
+
+	t.Run("Run inheritFlags()", func(t *testing.T) {
+		s := c.NewSubCommandInheritFlags("name", "description")
+		s.inheritFlags(c.flags)
```

Comment:
We should add a test that the flag is actually on `s` and `t.Fail()` if it is not 👍

Reply:
I've added tests for both `NewSubCommandInheritFlags()` and `inheritFlags()`